Feature work 4/2/22

toddtarsi opened the pull request:
Does the following:

Gets side-cli workin on a basic level and fully typed
Clears out extraneous package.json entries
Gets nav bar drag handling to be more consistent across platforms
Adds all core code-export packages back
Adds back and fully types side utils, renames to side-code-export

toddtarsi committed:
windows draggable and interactable nav
also, some misc cleanup in the package.json files

toddtarsi committed:
adding all code export libraries
and fully typing the side-runner

coinzdude commented:
I encountered this building:

```
yarn install v1.22.18
[1/5] Validating package.json...
[2/5] Resolving packages...
warning workspace-aggregator-0409156d-819c-463f-ab05-91e220a87f78 > @seleniumhq/side-runner > @types/winston@2.4.4: This is a stub types definition. winston provides its own type definitions, so you do not need this installed.
Couldn't find any versions for "@seleniumhq/side-utils" that matches "^4.0.0"
? Please choose a version of "@seleniumhq/side-utils" from this list: (Use arrow keys)
> 3.17.2
  3.17.0
  3.16.0
```

coinzdude commented:
I did select 3.17.2 and this builds and runs without error, and the draggable and resizable windows are very nice to have

toddtarsi committed:
fully typing the side-runner

toddtarsi committed:
tweak versions and adjust hoisting rules
chromedriver has to be off of the main path
or else side-runner tries to use it

toddtarsi committed:
get side playback working in nohoist environment

toddtarsi committed:
minor file lint

toddtarsi committed:
linting one more file
